### ISSUE TITLE
ArrayIndexOutOfBoundsException when reading split data

### ISSUE THREAD

**Issue by peternees** [sstrickx/yahoofinance-api#147]
getSplitHistory for SP.CN gives

<stderr>: java.lang.ArrayIndexOutOfBoundsException: 1
<stderr>: 	at yahoofinance.histquotes2.HistSplitsRequest.parseCSVLine(HistSplitsRequest.java:127)
<stderr>: 	at yahoofinance.histquotes2.HistSplitsRequest.getResult(HistSplitsRequest.java:116)
<stderr>: 	at yahoofinance.Stock.getSplitHistory(Stock.java:473)
<stderr>: 	at yahoofinance.Stock.getSplitHistory(Stock.java:454)

**Comment by antoinevulcain** [comment]
Yes just replace in line 124:

String[] parts = data[1].split(":");

**Comment by peternees** [comment]
It's a pitty this was not fixed in the new release. Now I keep needing my own version.

```
histquotes2.HistSplitsRequest
private HistoricalSplit parseCSVLine(String line) {
...
        String[] parts = data[1].split(":");
...
    }
```

**Comment by sstrickx** [comment]
I'll check it out this evening and provide a new release.

**Comment by sstrickx** [comment]
Fixed in version 3.17.0
